ext/uri/config.w32

```diff
@@ -0,0 +1,9 @@
+EXTENSION("uri", "php_uri.c", false /* never shared */, "/I ext/lexbor /I ext/uri/uriparser/include /DZEND_ENABLE_STATIC_TSRMLS_CACHE=1");
+
+AC_DEFINE("URI_ENABLE_ANSI", 1, "Define to 1 for enabling ANSI support of uriparser.")
+AC_DEFINE("URI_NO_UNICODE", 1, "Define to 1 for disabling unicode support of uriparser.")
+ADD_FLAG("CFLAGS_URI", "/D URI_LIBRARY_BUILD");
```

Comment:
You should be using URI_STATIC_BUILD rather than URI_LIBRARY_BUILD. That way, you don't create dll exports and link against the uriparser lib statically, which makes sense as you only use the lib directly in ext/uri. Furthermore, this flag would ideally also be passed in the m4 file for consistency, even though it would not change anything for that kind of build.

Reply:
Oh yes, thanks for answering my unspoken question :) I noticed `URI_STATIC_BUILD`, but wasn't sure if I should use it or not.. so at last I decided to go with `URI_LIBRARY_BUILD` following Lexbor's equivalent `LEXBOR_BUILDING` that was used in ext/lexbor, but obviously I missed the difference in their use-case (lexbor is shared between extensions vs uriparser is only used within ext/uri).